### ISSUE TITLE
Display issue in APC/Air Alarm program

### ISSUE THREAD

**Issue by GlowyFox** [CHOMPStation2/CHOMPStation2#452]
When using a laptop or tablet to access the map showing the location of APCs/air monitors, the icons are not displayed in-place - Rather, they are diagonally up and to the right, which makes using that feature rather impossible.

Attached is a screenshot of the issue with the air alarms, deck one. It is the same for that and the APC grid, and I think the issue happens for the console versions as well.

![image](https://user-images.githubusercontent.com/17265355/89604301-d3f73200-d81f-11ea-88df-321493a1dc54.png)

**Comment by Kazzc** [comment]
I think I know how to fix this, I may test and see if I can fix it. It is not a guaranteed

**Comment by Kazzc** [comment]
Yeah, nevermind. I hunch I tested is not what controls NanoUI-Map. I have no idea how to fix it... yet.

**Comment by Shadow-Quill** [comment]
This issue will likely be different/gone once https://github.com/PolarisSS13/Polaris/pull/7687 is merged.

**Comment by Raeschen** [comment]
no longer relevant due to UI changes
